qa/tasks/keycloak.py

```diff
@@ -273,9 +273,35 @@ def run_admin_cmds(ctx,config):
 
         ans6=out6.rstrip()
 
+        out7= toxvenv_sh(ctx, remote,
```

Comment:
Check if it is possible to parse 'sub', 'azp' in the output of 'aud' itself, because otherwise you are sending two extra same introspect curl commands which gives back the same response.